File: benchmarks/fib.go
package main

import (
	"fmt"
	"time"
)

func fib(n int) int {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	t0 := time.Now()
	fmt.Println(fib(35))
	fmt.Printf("bench_ns=%d\n", time.Since(t0).Nanoseconds())
}

File: benchmarks/fib.rs
fn fib(n: i64) -> i64 {
    if n < 2 {
        n
    } else {
        fib(n - 1) + fib(n - 2)
    }
}

fn main() {
    let t0 = std::time::Instant::now();
    let res = fib(35);
    let duration = t0.elapsed();
    println!("{}", res);
    println!("bench_ns={}", duration.as_nanos());
}
